cob_moveit_config/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(cob_moveit_config)
+
+find_package(catkin REQUIRED)
+
+catkin_package()
+
+set(robotlist
+    cob3-2
+    cob3-6
+    cob3-9
+    cob4-1
+    cob4-2
+    cob4-7
```

Comment:
will do this in a new PR when I update all moveit_configs according to latest cob_hardware_config...also including the new mesh https://github.com/ipa320/cob_manipulation/issues/89